schema/swagger-extensions.json

```diff
@@ -1589,6 +1592,51 @@
         "enum": ["create", "read", "update"]
       },
       "uniqueItems": true
+    },
+    "xmsResponseExample": {
+      "type": "object",
+      "description": "Provides the format of specifying an example response in the x-ms-examples extension.",
+      "properties": {
+        "headers": {
+          "type": "object",
+          "description": "Examples of response headers like 'location', 'azure-asyncoperation', 'retry-after', etc.",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "body":{
+          "type": ["object", "number", "array", "integer", "string", "boolean", "null"],
+          "description": "Example of the response body if applicable for the operation."
+        }
+      }
+    },
+    "xmsExampleObject": {
+      "type": "object",
+      "required": ["parameters", "responses"],
+      "description": "Describes the format of an example defined in the x-ms-examples extension.",
+      "properties": {
+        "parameters": {
+          "type": "object",
+          "description": "A dictionary of parameter names and their sample values for a particular operation.",
+          "additionalProperties": {
```

Comment:
The schema is a static thing. I do not know what kind of parameters would one want to provide. I mean, we surely know some common one's like subscriptionId, api-version, resourceGroupName. But these common one's are not always present as well. AD Graph does not have resourceGroupName and subscriptionId. 

We can go either way:
1 - The current approach that is taken in the PR
2 - Model the commonly used parameters as optional properties and set additionalProperties to true (so people can add more parameters).

What do you think?